### PR TITLE
User Interface Setup

### PR DESCRIPTION
**Pull request by bengigo**
**Changes made in this branch:**

- Create the first component to create the calculator
- Implement a basic UI style without functionalities

(✨no functionality is required for this stage of the project✨)

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+function Calculator() {
```
Comment (medaminedev66) [inline src/components/Calculator.js]: - Following the project requirements, the `Calculator.js` component should be a class-based component. Please try to refactor your component to a class-based component.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @bengigo  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional.
- GitFlow best practices have been followed
- There are no linter errors
- You have a descriptive PR

## Required Changes ♻️
- [x] There is still some boilerplate code that needs to be deleted from your repo. Please remove all unused files.

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+//       <div className="num cell">1</div>
+//       <div className="num cell">2</div>
+//       <div className="num cell">3</div>
+//       <div className="operand cell">+</div>
+//       <div className="num zero cell">0</div>
+//       <div className="point cell">.</div>
+//       <div className="equals operand cell">=</div>
+//     </div>
+//   );
+// }
```
Comment (youmari) [inline src/components/Calculator.js]: - [x] It would be better if you remove the unused commented code in order to keep your file clean and pro

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @bengigo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ PRO PR
✔️ Git flow
✔️ No linter error



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @bengigo,

## APPROVED 🟢

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**